### PR TITLE
Current info in placeholders is represented

### PR DESCRIPTION
**Pull request by PijusOz**
png-clipart-face-emoticon-and-rope-emoji-t-shirt-thought-computer-icons-fruit-linking-thinking-man-people-smiley

### CODE REVIEW

File: front-end-side/src/Pages/EnterData.js [hunk 1]
```diff
@@ -86,6 +92,7 @@ const ShowData = props => {
         // props.index - id, kuri keisti
         // purpose, amount, seller, date - jau update inpute ivesti duomenys
         axios.put("https://localhost:7174/api/ShowData/" + props.index, { "date": date, "seller": seller, "amount": amount, "purpose": purpose, "id": props.index });
+        document.getElementById("date").placeholder = date;
```
Comment (dissco6) [inline front-end-side/src/Pages/EnterData.js:95]: Irrelevant

Comment (dissco6) [inline front-end-side/src/Pages/EnterData.js:95]: Irrelevant

File: back-end-side/Controllers/ShowDataController.cs [hunk 2]
```diff
@@ -31,11 +31,7 @@ public IActionResult Put([FromBody] Record model, int id)
         {
             // pirma - nustatom kategorija, tada pakeiciam tam tikru indexo value ir tada sortinam
             model.Category = Sorting.CheckInput(model);
-            int index = UploadController.RecordsFromFile.FindIndex(ind => ind.Equals(id));
-            if (index == -1)
```
Comment (shilingas) [inline back-end-side/Controllers/ShowDataController.cs:35]: why you deleted if statement?

File: front-end-side/src/Pages/EnterData.js (hunk 1 above)
Comment (PijusOz) [reply front-end-side/src/Pages/EnterData.js:95]: fair enough

File: back-end-side/Controllers/ShowDataController.cs (hunk 2 above)
Comment (PijusOz) [reply back-end-side/Controllers/ShowDataController.cs:35]: why you wirited if statement?

Comment (dissco6) [reply back-end-side/Controllers/ShowDataController.cs:35]: what wirited means?

Comment (PijusOz) [reply back-end-side/Controllers/ShowDataController.cs:35]: FindIndex returns index if index is found (index = index returns index), and -1 in other cases. Here you were comparing object with int, object and int will never be equal, therefore it returns -1 all the time, so if statement and findIndex are irrelevant

**Pull request merged**